sftp/src/main/scala/blobstore/sftp/SftpStore.scala

```diff
@@ -19,24 +19,25 @@ package sftp
 import java.util.Date
 
 import com.jcraft.jsch._
+import cats.syntax.all._
 import cats.instances.option._
 
 import scala.util.Try
 import java.io.OutputStream
 
-import cats.Traverse
-import cats.effect.{Blocker, ConcurrentEffect, ContextShift, IO, Resource}
+import cats.Applicative
+import cats.effect.{Blocker, ConcurrentEffect, ContextShift, Effect, IO, Resource}
 import cats.effect.concurrent.{MVar, Semaphore}
 import fs2.concurrent.Queue
 
-final class SftpStore[F[_]](
+final class SftpStore[F[_]: ConcurrentEffect](
   absRoot: String,
   private[sftp] val session: Session,
   blocker: Blocker,
   mVar: MVar[F, ChannelSftp],
   semaphore: Option[Semaphore[F]],
   connectTimeout: Int
-)(implicit F: ConcurrentEffect[F], CS: ContextShift[F]) extends Store[F] {
+)(implicit CS: ContextShift[F]) extends Store[F] {
```

Comment:
Consider keeping existing de-sugared syntax to be consistent with different strore types.
Same for the constructor in companion object.